### PR TITLE
deps: alpine 3.19.1

### PR DESCRIPTION
**Pull request by codefromthecrypt**
I also updated the description, but re-tagging the image will only update the alpine layer as latest JRE is the same build as before.

### CODE REVIEW

**Review by reta — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ ARG docker_parent_image=ghcr.io/openzipkin/alpine:3.19.0
 # When updating, also update the README
 #  * Use current version from https://pkgs.alpinelinux.org/packages?name=openjdk21, stripping
 #    the `-rX` at the end.
-ARG java_version=21.0.1_p12
+ARG java_version=21.0.2_p13
```
Comment (reta) [inline Dockerfile:20]: Seems like `21.0.2_p13-r2`?

Comment (codefromthecrypt) [reply Dockerfile:20]: see comment above the line? :D

**Comment by codefromthecrypt** [comment]
@reta hmm odd.. I wonder if there's an non-determinism issue in the package selection, as you can see *21.0.1* in the breaks part despite the ultimate expression used being `~openjdk21-jdk-21.0.2_p13` and `~openjdk21-jmods-21.0.2_p13` (where the tilde says get this prefix, which we need because it reduces maintenance when there's a package re-do)

```
 0.943 ERROR: unable to select packages:
0.945   openjdk21-jmods-21.0.2_p13-r2:
0.945     breaks: world[openjdk21-jmods~21.0.1_p12]
0.945     satisfies: openjdk21-jdk-21.0.2_p13-r2[openjdk21-jmods]
0.945   openjdk21-jdk-21.0.2_p13-r2:
0.945     breaks: world[openjdk21-jdk~21.0.1_p12]
```

**Comment by codefromthecrypt** [comment]
I think we don't need to separately add the jmods things, because it is already a dependency of the jdk
https://pkgs.alpinelinux.org/package/edge/community/x86_64/openjdk21-jdk Maybe this will reduce the issue

**Comment by codefromthecrypt** [comment]
can someone with intel machine, run from this branch `./build-bin/test 21.0.2_p13`? from my macbook zero issues, but obviously something is happening here on x86, and not sure if there's some change in the runner or if something else is at play

File: Dockerfile (hunk 1 above)
Comment (reta) [reply Dockerfile:20]: Or sorry, my bad

**Comment by reta** [comment]
> @reta hmm odd.. I wonder if there's an non-determinism issue in the package selection, as you can see _21.0.1_ in the breaks part despite the ultimate expression used being `~openjdk21-jdk-21.0.2_p13` and `~openjdk21-jmods-21.0.2_p13` (where the tilde says get this prefix, which we need because it reduces maintenance when there's a package re-do)

Yeah, it confused (and derailed) me as well, not sure what is happening

**Comment by codefromthecrypt** [comment]
I think it is repository order, since openjdk is now not only in edge/community.. will do a quick change to verify, then maybe update the base layer

**Comment by reta** [comment]
> can someone with an x86 machine, run from this branch `./build-bin/test 21.0.2_p13`? from my macbook zero issues, but obviously something is happening here on x86, and not sure if there's some change in the runner or if something else is at play

No issues on Linux with  `./build-bin/test 21.0.2_p13` for me

**Comment by codefromthecrypt** [comment]
lol force-broken-world just doesn't do anything

**Comment by codefromthecrypt** [comment]
so this isn't about 3.19.1 as same error reverting to 3.19.0... there's something funny and I don't think it is the mirror as I already switched that. Maybe something changed in the runner 🤷

**Comment by codefromthecrypt** [comment]
ok it had to do with the github workflow run. basically it ran install twice, once with a slightly older patch and then again with the current one. I guess the change in build-args isn't tainting the layers?

**Comment by codefromthecrypt** [comment]
ok good to go. sorry about the thrash, I forgot build-arg cache invalidation behavior

**Pull request merged**